scripts-available/CDB_FederatedServerDiagnostics.sql

```diff
@@ -98,21 +98,119 @@ $$
 LANGUAGE SQL VOLATILE PARALLEL UNSAFE;
 
 
+--
+-- Get a foreign PG server hostname from the catalog
+--
+CREATE OR REPLACE FUNCTION @extschema@.__CDB_FS_Foreign_Server_Host_PG(server_internal name)
```

Comment:
Would it make sense to unify `__CDB_FS_Foreign_Server_Host_PG` and `__CDB_FS_Foreign_Server_Port_PG` into a common function that receives the server name and the name of the property that we want to extract?

Reply:
I will postpone this to a future iteration.

Another idea for the future: move `__CDB_FS_Import_If_Not_Exists` and simplify code in other files.